todo_app/data/session_items.py

```diff
@@ -26,6 +26,7 @@ def get_item(id):
 
     Args:
```

Comment:
I don't think you're using this functionality at all any more, so we can get rid of it